package.nls.json

```diff
@@ -36,6 +36,7 @@
     "AWS.command.deleteLambda": "Delete",
     "AWS.command.deleteLambda.confirm": "Are you sure you want to delete lambda function '{0}'?",
     "AWS.command.deleteLambda.error": "There was an error deleting lambda function '{0}'",
+    "AWS.command.addSamDebugConfiguration": "Add Debug Config",
```

Comment:
Yeah I just abbreviated because it seemed really long. Think this is fine?

![image](https://user-images.githubusercontent.com/1410453/77490591-85a93f80-6df8-11ea-8612-8a1efe548b60.png)

Reply:
That looks great to me -- it is contextually relevant, and only shows up in template files that follow the SAM/CloudFormation structure